wasmsdk/README.md

```diff
@@ -183,6 +183,51 @@ updates allocation settings
 > hash: string
 
 
+### zcn.sdk.getAllocationFromAuthTicket
+returns allocation from authToken
+
+**Input**:
+> authTicket string
+
+**Output**:
+> [sdk.Allocation](https://github.com/0chain/gosdk/blob/a9e504e4a0e8fc76a05679e4ef183bb03b8db8e5/zboxcore/sdk/allocation.go#L140)
+
+### zcn.sdk.getReadPoolInfo
+gets information about the read pool for the allocation
+
+**Input**:
+> clientID string
+
+**Output**:
+> [sdk.ReadPool](https://github.com/0chain/gosdk/blob/6878504e4e4d7cb25b2ac819c3c578228b3d3e30/zboxcore/sdk/sdk.go#L167-L169)
+
+
+### zcn.sdk.redeemFreeStorage
+
+**Input**:
+> ticket string
+
+**Output**:
+> hash string
+
+### zcn.sdk.decodeTicket
+
+**Input**:
+> ticket string
+
+**Output**:
+> recipientPublicKey string, markerStr string, tokensInSAS uint64
+
+
+### zcn.sdk.writePoolLock
```

Comment:
is it better to use `lockWritePool` or `lockTokensInWritePool` ?

Reply:
fixed in ee7cc18